internal/platform/ipu.go

```diff
@@ -87,7 +87,9 @@ func (pi *IntelDetector) IsDpuPlatform(platform Platform) (bool, error) {
 func (pi *IntelDetector) GetDpuIdentifier(platform Platform, pci *ghw.PCIDevice) (plugin.DpuIdentifier, error) {
 	// TODO: rethink if it's possible to use something else than a pci address. Serial number doesn't seem to be the
 	// right choice for IPU.
-	identifier := fmt.Sprintf("intel-ipu-%s", pci.Address)
+	// Sanitize PCI address for Kubernetes resource name (replace : with -)
```

Comment:
Is this unique for IPU? Or can it be applied for Marvell?
For PTL we use the serial number.